charts/gsp-cluster/templates/02-gsp-system/calico.yaml

```diff
@@ -500,9 +506,10 @@ spec:
       labels:
         k8s-app: calico-typha
       annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
         cluster-autoscaler.kubernetes.io/safe-to-evict: 'true'
     spec:
-      priorityClassName: system-cluster-critical
+      priorityClassName: system-node-critical
```

Comment:
`calico-typha` probably doesn't need to be `node-critical` since it's a Deployment not a DaemonSet